db/schema.rb

```diff
@@ -252,6 +253,39 @@
     t.datetime "updated_at"
   end
 
+  create_table "tasks", force: :cascade do |t|
+    t.string "title"
+    t.text "description"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.boolean "finished"
+    t.date "date"
+    t.integer "total_no_of_days"
+    t.integer "no_of_days_followed"
+  end
+
+  create_table "todo_items", force: :cascade do |t|
+    t.string "content"
+    t.bigint "todo_list_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["todo_list_id"], name: "index_todo_items_on_todo_list_id"
+  end
+
+  create_table "todo_lists", force: :cascade do |t|
+    t.date "date"
+    t.boolean "finished"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "todolists", force: :cascade do |t|
```

Comment:
I'm noticing there is both `todo_lists` and `todolists`. You might have to rollback your migrations to remove the unwanted `todolists`.